test/lib/ufe/testMoveCmd.py

```diff
@@ -276,6 +281,118 @@ def ufeSceneItemTranslation(item):
 
         self.runMultiSelectTestMove(ballItems, expected)
 
+    def runTestOpUndo(self, createTransformOp, attrSpec):
```

Comment:
TDD test: wrote the test first, the matrix op and common API undo / redo failed, fixed the implementation, test now passes.